src/View/Form/ArrayContext.php

```diff
@@ -80,9 +73,9 @@ class ArrayContext implements ContextInterface
      * @param \Cake\Network\Request $request The request object.
      * @param array $context Context info.
      */
-    public function __construct(Request $request, array $context)
+    public function __construct(Request $request, array $context, $requestType)
```

Comment:
@markstory Adding new argument without default value is against out backwards compatibility guidelines right?